lib/install.js

```diff
@@ -103,7 +110,7 @@ module.exports = async function(version) {
 
   mkdirp.sync(path.join(common.getNvmCacheDir(), version));
 
-  const status = await downloadNode(version);
+  const status = await downloadNode(version,proxy);
```

Comment:
need to pass the `verifyssl` param

Reply:
forget to include it in this branch, great catch thanks!

Reply:
added